docs/_newsfragments/777.breakingchange.rst

```diff
@@ -1,2 +1,2 @@
-The class :class:`~.falcon.http_error.NoRepresentation` was removed since
-all :class:`~.falcon.HTTPError` have optional representation.
\ No newline at end of file
+The class :class:`~.falcon.http_error.NoRepresentation` was removed. All
```

Comment:
Same as above, the breaking change is too abrupt IMHO.

Reply:
This was removed in https://github.com/falconry/falcon/pull/1655

Reply:
corrected to deprecated, since it was readded